addons/survey/views/survey_templates_management.xml

```diff
@@ -110,7 +110,7 @@
     </template>
 
     <template id="survey_button_retake" name="Survey: retake button">
-        <div>
+        <div class="d-print-none">
             <t t-if="not answer.scoring_success and not answer.is_session_answer">
```

Comment:
Would put the t-if inside the div, currently it creates an empty div so the `gap-3` class is applied and you get a weird space if "take gain" doesn't appear.